src/resources/views/crud/fields/address_algolia.blade.php

```diff
@@ -54,6 +54,7 @@
 
     {{-- FIELD CSS - will be loaded in the after_styles section --}}
     @push('crud_fields_styles')
+    @stack('on_the_fly_styles')
```

Comment:
@pxpm I know you've already explained why we _need_ to add this Blade stack to ALL fields, but... I forgot 😄  That's why it's better to explain things in writing, rather than Skype, I guess.

**Can you please explain again, in writing, in as many words as possible 😄 ?** I really think we should find a way around this. Understanding _why_ we need this is the first step towards finding an alternative solution. 

There are many reasons why I don't agree with changing all fields to accommodate this one feature:
- it means the fields are no longer decoupled from operations; which is a reduction of our coding standards;
- it's a major breaking change - anybody who's ever overwritten a field has to do this; for all their fields; this has happened only once before, when upgrading from v3 to v4, but that was a Major version update;
- it forces everybody else who creates a custom field to also do this, to support the operation; so it not only bloats the current fields for that one operation, but it also bloats all future fields;
- it creates a really bad precedent - that we can change A LOT of files to accommodate one feature; do it once, and you'll do it again; and again; before you know it, all your files are interconnected and when you change one small thing everything breaks; this is exaggerated a bit, but you understand what I mean - it's really easy to get to that point;

That's what I'm saying. Let's better understand what the need behind this is. And brainstorm an alternative solution.

Reply:
You are right. After going back and forth I introduced myself a problem the way I was pushing the styles/scripts. just to not override `show_fields.blade.php`

I did it, and fields does not need the stack anymore :) Problem solved.

Reply:
Done, removed field changes ;) It's 100% BC shielded i think!